clustering/common/src/main/java/org/jboss/as/clustering/controller/CommonRequirement.java

```diff
@@ -38,6 +38,7 @@
     ELYTRON("org.wildfly.security.elytron", Void.class),
     LOCAL_TRANSACTION_PROVIDER("org.wildfly.transactions.global-default-local-provider", Void.class),
     MBEAN_SERVER("org.wildfly.management.jmx", MBeanServer.class),
+    MICROPROFILE_CONFIG("org.wildfly.microprofile.config", Void.class),
```

Comment:
This doesn't appear to be used anywhere here, perhaps accidental?

Reply:
Ah yes - that was a remnant from a previous 10.1.x release where MP config was required.  We can remove this now.